### PR TITLE
Updated console logs to appropriate log level

### PR DESCRIPTION
**Pull request by leobcgov**
# Fixes or Implements [AB#8778](https://qslvic.visualstudio.com/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/8778)

* [X] Enhancement
* [ ] Bug
* [ ] Documentation

## Reviewer Guidelines

A reviewer should examine the PR if the Status above indicates the PR is **Ready**.
Once ready, the reviewer should read the information below as well as supporting information like the Azure ticket to ensure they have a good understanding of the request.

The reviewer should then:

1. Is the PR template complete and accurate?

    Confirm by reading all sections below.

1. Confirm the merge branch

    A normal feature/bug branch should merge into the Dev branch.  Exceptions for hotfix and sprint merges do occur and the target would then be Master.

1. Do the code commits have an appropriate comment and include linkage to the azure ticket by including a reference to "AB#nnnn"? 

1. Review the Code Changes

    1. Do the changes match the description?

        The code commits should reflect the documented description and nothing more.  If additional changes are included the reviewer should call these out and request that the developer document the changes in Azure and in the description or remove the changed code.

    1. useful tips

        * Any obvious errors or omissions?
        * Does the code match the style of other similar work?
        * Is the code easy to understand?
        * Is the code well documented?
        * DB Migrations are appropriate?

If the Answer to any of the questions above is No, then a request for change should be the result of the review.

## Description
- replace all the console.log with winston.logger at info/warn or error level

## Testing

* [ ] Unit Tests Updated
* [ ] Functional Tests Updated
* [X] Not Required

### Steps to Reproduce:

If this is a bug, please provide details on how to reproduce the code.

### UI Changes

NO

### Browsers Tested

* [X] Chrome
* [ ] Safari
* [ ] Edge
* [ ] Firefox

Provide an explanation for any test(s) not completed.

## Notes/Additional Comments

Any additional notes or comments that may be relevant.  Include any specialized deployment steps here.

### CODE REVIEW

File: Apps/WebClient/src/ClientApp/src/services/winstonLogger.ts [hunk 1]
```diff
@@ -7,12 +7,9 @@ import { ExternalConfiguration } from "@/models/configData";
 export class WinstonLogger implements ILogger {
     private logger!: Logger;
 
-    public initialize(config: ExternalConfiguration): void {
+    public initialize(logLevel?: string): void {
```
Comment (leobcgov) [inline Apps/WebClient/src/ClientApp/src/services/winstonLogger.ts:10]: updated the input param to ease the log settings for vue's jest test scripts

File: Apps/WebClient/src/ClientApp/src/components/navmenu/sidebar.vue [hunk 2]
```diff
@@ -606,7 +607,7 @@ export default class SidebarComponent extends Vue {
     }
 
     private dismissTutorial() {
-        console.log("Dismissing tutorial...");
+        this.logger.info("Dismissing tutorial...");
```
Comment (FuriousLlama) [inline Apps/WebClient/src/ClientApp/src/components/navmenu/sidebar.vue:610]: debug lvl, the user would get much out of this

File: Apps/WebClient/src/ClientApp/src/components/timeline/medication.vue [hunk 3]
```diff
@@ -253,7 +257,9 @@ export default class MedicationTimelineComponent extends Vue {
         // Load medication details
         if (!this.medicationLoaded) {
             this.isLoadingMedication = true;
-            console.log(medicationEntry);
+            this.logger.info(
+                `Loading Medication Entry : ${JSON.stringify(medicationEntry)}`
```
Comment (FuriousLlama) [inline Apps/WebClient/src/ClientApp/src/components/timeline/medication.vue:261]: on the fence on this one; could be dbg

File: Apps/WebClient/src/ClientApp/src/services/httpDelegate.ts [hunk 4]
```diff
@@ -74,14 +78,14 @@ export default class HttpDelegate implements IHttpDelegate {
             const config: AxiosRequestConfig = {
                 headers,
             };
-            console.log("Config:", config);
+            this.logger.info(`Config: ${JSON.stringify(config)}`);
```
Comment (FuriousLlama) [inline Apps/WebClient/src/ClientApp/src/services/httpDelegate.ts:81]: dbg?

File: Apps/WebClient/src/ClientApp/src/services/httpDelegate.ts [hunk 5]
```diff
@@ -96,14 +100,14 @@ export default class HttpDelegate implements IHttpDelegate {
...
```
Comment (FuriousLlama) [inline Apps/WebClient/src/ClientApp/src/services/httpDelegate.ts:103]: debug level

File: Apps/WebClient/src/ClientApp/src/services/restAuthService.ts [hunk 6]
```diff
@@ -47,13 +54,13 @@ export class RestAuthenticationService implements IAuthenticationService {
             loadUserInfo: false,
             automaticSilentRenew: true,
         };
-        console.log("oidc configuration: ", oidcConfig);
+        this.logger.info(`oidc configuration: ${JSON.stringify(oidcConfig)}`);
```
Comment (FuriousLlama) [inline Apps/WebClient/src/ClientApp/src/services/restAuthService.ts:57]: debug

File: Apps/WebClient/src/ClientApp/src/services/restMedicationService.ts [hunk 7]
```diff
@@ -57,7 +64,9 @@ export class RestMedicationService implements IMedicationService {
                     resolve(requestResult);
                 })
                 .catch((err) => {
-                    console.log(this.FETCH_ERROR + err.toString());
+                    this.logger.info(
```
Comment (FuriousLlama) [inline Apps/WebClient/src/ClientApp/src/services/restMedicationService.ts:67]: error

File: Apps/WebClient/src/ClientApp/src/services/restMedicationService.ts [hunk 8]
```diff
@@ -75,7 +84,9 @@ export class RestMedicationService implements IMedicationService {
                     this.handleResult(requestResult, resolve, reject);
                 })
                 .catch((err) => {
-                    console.log(this.FETCH_ERROR + err.toString());
+                    this.logger.info(
```
Comment (FuriousLlama) [inline Apps/WebClient/src/ClientApp/src/services/restMedicationService.ts:87]: error

File: Apps/WebClient/src/ClientApp/src/services/restUserNoteService.ts [hunk 9]
```diff
@@ -51,7 +58,7 @@ export class RestUserNoteService implements IUserNoteService {
     NOT_IMPLENTED: string = "Method not implemented.";
 
     public createNote(note: UserNote): Promise<UserNote> {
-        console.log(note);
+        this.logger.info(`createNote: ${JSON.stringify(note)}`);
```
Comment (FuriousLlama) [inline Apps/WebClient/src/ClientApp/src/services/restUserNoteService.ts:61]: debug or trace

File: Apps/WebClient/src/ClientApp/src/store/modules/auth/actions.ts [hunk 10]
```diff
@@ -32,14 +37,14 @@ export const actions: ActionTree<AuthState, RootState> = {
...
-                    console.log("Could not get the user!");
+                    logger.warn("Could not get the user!");
                     context.dispatch("clearStorage");
                     hasAccess = false;
                 } else {
                     context.dispatch("oidcWasAuthenticated", oidcUser);
                     if (!isAuthenticatedInStore) {
                         // We can add events when the user wasnt authenticated and now it is.
-                        console.log(
+                        logger.info(
```
Comment (FuriousLlama) [inline Apps/WebClient/src/ClientApp/src/store/modules/auth/actions.ts:47]: debug or trace

File: Apps/WebClient/src/ClientApp/src/store/modules/auth/mutations.ts [hunk 11]
```diff
@@ -1,11 +1,16 @@
...
+import { SERVICE_IDENTIFIER } from "@/plugins/inversify";
+import { ILogger } from "@/services/interfaces";
 import { User as OidcUser } from "oidc-client";
 
+const logger: ILogger = container.get(SERVICE_IDENTIFIER.Logger);
+
 export const mutations: MutationTree<AuthState> = {
     setOidcAuth(state: AuthState, user: OidcUser) {
-        console.log("setOidcAuth");
+        logger.info(`setOidcAuth}`);
```
Comment (FuriousLlama) [inline Apps/WebClient/src/ClientApp/src/store/modules/auth/mutations.ts:13]: trace

Comment (FuriousLlama) [reply Apps/WebClient/src/ClientApp/src/store/modules/auth/mutations.ts:13]: all others are trace as well

File: Apps/WebClient/src/ClientApp/src/store/modules/laboratory/actions.ts [hunk 12]
```diff
@@ -2,13 +2,15 @@ import { ActionTree, Commit } from "vuex";
...
+import { ILogger, ILaboratoryService } from "@/services/interfaces";
 import { LaboratoryOrder } from "@/models/laboratory";
 import RequestResult from "@/models/requestResult";
 import { ResultType } from "@/constants/resulttype";
 
+const logger: ILogger = container.get(SERVICE_IDENTIFIER.Logger);
+
 function handleError(commit: Commit, error: Error) {
-    console.log("ERROR:" + error);
+    logger.info(`ERROR: ${JSON.stringify(error)}`);
```
Comment (FuriousLlama) [inline Apps/WebClient/src/ClientApp/src/store/modules/laboratory/actions.ts:13]: error

File: Apps/WebClient/src/ClientApp/src/store/modules/laboratory/actions.ts [hunk 13]
```diff
@@ -24,7 +26,7 @@ export const actions: ActionTree<LaboratoryState, RootState> = {
         return new Promise((resolve, reject) => {
             const laboratoryOrders: LaboratoryOrder[] = context.getters.getStoredLaboratoryOrders();
             if (laboratoryOrders.length > 0) {
-                console.log("Laboratory found stored, not quering!");
+                logger.info(`Laboratory found stored, not quering!`);
```
Comment (FuriousLlama) [inline Apps/WebClient/src/ClientApp/src/store/modules/laboratory/actions.ts:29]: debug

File: Apps/WebClient/src/ClientApp/src/store/modules/laboratory/actions.ts [hunk 14]
```diff
@@ -34,7 +36,7 @@ export const actions: ActionTree<LaboratoryState, RootState> = {
                     totalResultCount: laboratoryOrders.length,
                 });
             } else {
-                console.log("Retrieving Laboratory Orders");
+                logger.info(`Retrieving Laboratory Orders`);
```
Comment (FuriousLlama) [inline Apps/WebClient/src/ClientApp/src/store/modules/laboratory/actions.ts:39]: debug

File: Apps/WebClient/src/ClientApp/src/store/modules/medication/actions.ts [hunk 15]
```diff
@@ -22,11 +23,10 @@ export const actions: ActionTree<MedicationState, RootState> = {
                 params.din
             );
             if (medicationResult) {
-                console.log("Medication found stored, not quering!");
-                //console.log("Medication Data: ", medicationResult);
+                logger.info(`Medication found stored, not quering!`);
                 resolve(medicationResult);
             } else {
-                console.log("Retrieving Medication info");
+                logger.info(`Retrieving Medication info...`);
```
Comment (FuriousLlama) [inline Apps/WebClient/src/ClientApp/src/store/modules/medication/actions.ts:29]: debug?

File: Apps/WebClient/src/ClientApp/src/store/modules/medication/getters.ts [hunk 16]
```diff
@@ -1,14 +1,18 @@
...
+const logger: ILogger = container.get(SERVICE_IDENTIFIER.Logger);
+
 export const getters: GetterTree<MedicationState, RootState> = {
     getStoredMedication: (state: MedicationState) => (
         din: string
     ): MedicationResult | undefined => {
         din = din.padStart(8, "0");
-        console.log(din);
-
+        logger.info(`getStoredMedication: din = ${JSON.stringify(din)}`);
```
Comment (FuriousLlama) [inline Apps/WebClient/src/ClientApp/src/store/modules/medication/getters.ts:15]: debug

File: Apps/WebClient/src/ClientApp/src/store/modules/sidebar/mutations.ts [hunk 17]
```diff
@@ -1,13 +1,18 @@
...
+import { SERVICE_IDENTIFIER } from "@/plugins/inversify";
+import { ILogger } from "@/services/interfaces";
 import { SidebarState } from "@/models/storeState";
 
+const logger: ILogger = container.get(SERVICE_IDENTIFIER.Logger);
+
 export const mutations: MutationTree<SidebarState> = {
     toggle(state: SidebarState) {
-        console.log("SidebarState:toggle");
+        logger.info(`SidebarState:toggle`);
```
Comment (FuriousLlama) [inline Apps/WebClient/src/ClientApp/src/store/modules/sidebar/mutations.ts:11]: trace

File: Apps/WebClient/src/ClientApp/src/store/modules/sidebar/mutations.ts [hunk 18]
```diff
@@ -1,13 +1,18 @@
...
         state.isOpen = !state.isOpen;
     },
     setState(state: SidebarState, isOpen: boolean) {
-        console.log("SidebarState:setState");
+        logger.info(`SidebarState:setState`);
```
Comment (FuriousLlama) [inline Apps/WebClient/src/ClientApp/src/store/modules/sidebar/mutations.ts:15]: trace

File: Apps/WebClient/src/ClientApp/src/store/modules/user/actions.ts [hunk 19]
```diff
@@ -27,7 +33,7 @@ export const actions: ActionTree<UserState, RootState> = {
             patientService
                 .getPatientData(params.hdid)
                 .then((patientData) => {
-                    console.log("Patient Data: ", patientData);
+                    logger.info(`Patient Data: ${JSON.stringify(patientData)}`);
```
Comment (FuriousLlama) [inline Apps/WebClient/src/ClientApp/src/store/modules/user/actions.ts:36]: debug, throughout the file

File: Apps/WebClient/src/ClientApp/src/store/modules/user/mutations.ts [hunk 20]
```diff
@@ -43,7 +47,7 @@ export const mutations: MutationTree<UserState> = {
             "preferences",
             userProfile ? userProfile.preferences : {}
         );
-        console.log(state.user);
+        logger.info(`state.user: ${JSON.stringify(state.user)}`);
```
Comment (FuriousLlama) [inline Apps/WebClient/src/ClientApp/src/store/modules/user/mutations.ts:50]: debug

File: Apps/WebClient/src/ClientApp/src/store/modules/user/mutations.ts [hunk 21]
```diff
@@ -78,7 +82,11 @@ export const mutations: MutationTree<UserState> = {
         state: UserState,
         preference: { name: string; value: string }
     ) {
-        console.log(preference.name, preference.value);
+        logger.info(
+            `setUserPreference: preference.name: ${JSON.stringify(
```
Comment (FuriousLlama) [inline Apps/WebClient/src/ClientApp/src/store/modules/user/mutations.ts:86]: debug

File: Apps/WebClient/src/ClientApp/src/views/profile.vue [hunk 22]
```diff
@@ -561,7 +563,9 @@ export default class ProfileView extends Vue {
                 if (results[3]) {
                     // Load user profile
                     this.userProfile = results[3];
-                    console.log("User Profile: ", this.userProfile);
+                    this.logger.info(
+                        `User Profile: ${JSON.stringify(this.userProfile)}`
```
Comment (FuriousLlama) [inline Apps/WebClient/src/ClientApp/src/views/profile.vue:567]: debug

File: Apps/WebClient/src/ClientApp/src/views/profile.vue [hunk 23]
```diff
@@ -709,9 +712,7 @@ export default class ProfileView extends Vue {
             this.submitStatus = "ERROR";
         } else {
             this.submitStatus = "PENDING";
-
-            console.log(this.email);
-
+            this.logger.info(`saveEmailEdit: ${JSON.stringify(this.email)}`);
```
Comment (FuriousLlama) [inline Apps/WebClient/src/ClientApp/src/views/profile.vue:715]: debug

File: Apps/WebClient/src/ClientApp/src/views/profile.vue [hunk 24]
```diff
@@ -744,7 +745,7 @@ export default class ProfileView extends Vue {
             emailAddress: this.email,
         })
             .then(() => {
-                console.log("success!");
+                this.logger.info("success!");
```
Comment (FuriousLlama) [inline Apps/WebClient/src/ClientApp/src/views/profile.vue:748]: trace or not necessary

File: Apps/WebClient/src/ClientApp/src/views/profile.vue [hunk 25]
```diff
@@ -754,16 +755,16 @@ export default class ProfileView extends Vue {
...
             })
             .finally(() => {
                 this.isLoading = false;
             });
     }
 
     private updateSMS(): void {
-        console.log(
-            "Updating " + this.smsNumber ? this.smsNumber : "sms number..."
+        this.logger.info(
```
Comment (FuriousLlama) [inline Apps/WebClient/src/ClientApp/src/views/profile.vue:766]: debug

File: Apps/WebClient/src/ClientApp/src/views/profile.vue [hunk 26]
```diff
@@ -797,11 +798,11 @@ export default class ProfileView extends Vue {
             hdid: this.user.hdid,
         })
             .then(() => {
-                console.log("success!");
+                this.logger.info("success!");
```
Comment (FuriousLlama) [inline Apps/WebClient/src/ClientApp/src/views/profile.vue:801]: debug or trace

File: Apps/WebClient/src/ClientApp/src/views/profile.vue [hunk 27]
```diff
@@ -822,12 +823,12 @@ export default class ProfileView extends Vue {
...
```
Comment (FuriousLlama) [inline Apps/WebClient/src/ClientApp/src/views/profile.vue:826]: debug or trace

File: Apps/WebClient/src/ClientApp/src/views/registration.vue [hunk 28]
```diff
@@ -444,7 +446,11 @@ export default class RegistrationView extends Vue {
                     this.betaRequestService
                         .getRequest(this.oidcUser.hdid)
                         .then((betaRequest) => {
-                            console.log("beta request:", betaRequest);
+                            this.logger.info(
```
Comment (FuriousLlama) [inline Apps/WebClient/src/ClientApp/src/views/registration.vue:449]: debug

File: Apps/WebClient/src/ClientApp/src/views/registration.vue [hunk 29]
```diff
@@ -530,11 +536,13 @@ export default class RegistrationView extends Vue {
         this.userProfileService
             .getTermsOfService()
             .then((result) => {
-                console.log(result);
+                this.logger.info(
+                    `getTermsOfService result: ${JSON.stringify(result)}`
```
Comment (FuriousLlama) [inline Apps/WebClient/src/ClientApp/src/views/registration.vue:540]: debug

File: Apps/WebClient/src/ClientApp/src/views/registration.vue [hunk 30]
```diff
@@ -568,7 +576,9 @@ export default class RegistrationView extends Vue {
                     inviteCode: this.inviteKey || "",
                 })
                 .then((result) => {
-                    console.log(result);
+                    this.logger.info(
+                        `Create Profile result: ${JSON.stringify(result)}`
```
Comment (FuriousLlama) [inline Apps/WebClient/src/ClientApp/src/views/registration.vue:580]: debug

File: Apps/WebClient/src/ClientApp/src/views/termsOfService.vue [hunk 31]
```diff
@@ -84,11 +86,13 @@ export default class TermsOfServiceView extends Vue {
         this.userProfileService
             .getTermsOfService()
             .then((result) => {
-                console.log(result);
+                this.logger.info(
+                    "Terms Of Service retrieved: " + JSON.stringify(result)
```
Comment (FuriousLlama) [inline Apps/WebClient/src/ClientApp/src/views/termsOfService.vue:90]: debug

**Comment by leobcgov** [comment]
Thanks @FuriousLlama for pointing some mistakes in logging.

**Pull request merged**